packages/server/graphql/private/mutations/processRecurrence.ts

```diff
@@ -37,7 +38,8 @@ const startRecurringTeamPrompt = async (
   )
   const meeting = await safeCreateTeamPrompt(meetingName, teamId, facilitatorId, r, dataLoader, {
     scheduledEndTime: nextMeetingStartDate,
-    meetingSeriesId: meetingSeries.id
+    meetingSeriesId: meetingSeries.id,
+    meetingPrompt: lastMeeting ? lastMeeting.meetingPrompt : undefined
```

Comment:
+1 I wonder, how the `lastMeeting` can be null here? If recurrence is running, it means there's already at least one meeting and there will always be a prompt to reuse. 

If that's true, I think it'd be a good time to cleanup this, as it'd be used when first meeting is created. It'd be cleaner to have a const somewhere with default prompt and always provide it when creating an initial meeting ie. make `prompt` required parameter of `safeCreateTeamPrompt`.
https://github.com/ParabolInc/parabol/blob/287167069c78ac7f32b0f8ad9d0e05b020c5a410/packages/server/graphql/mutations/helpers/safeCreateTeamPrompt.ts#L40 

Also, if the `lastMeeting` is `null` in the case above, we'd override the prompt provided by constructor with `undefined`. So it kinda works, because it'll never happen

Reply:
There _should_ always be a `lastMeeting`, but there are no DB constraints that enforce this, so it's not guaranteed.

I extracted the default prompt and used that instead of `undefined`.